### PR TITLE
Add tests for callback_shortcuts.0.dart API example.

### PR DESCRIPTION
**Pull request by ksokolovskyi**
This PR contributes to https://github.com/flutter/flutter/issues/130459

### Description
- Corrects the name for `examples/api/test/widgets/shortcuts/callback_shortcuts.0_test.dart` test file
- Update tests for `examples/api/lib/widgets/shortcuts/callback_shortcuts.0.dart`

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
- [X] All existing and new tests are passing.

### CODE REVIEW

**Review by bleroux — APPROVED** [review]
LGTM! Thanks for adding this missing test.

File: examples/api/test/widgets/shortcuts/callback_shortcuts.0_test.dart [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_api_samples/widgets/shortcuts/callback_shortcuts.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('CallbackShortcutsApp', () {
```
Comment (TahaTesser) [inline examples/api/test/widgets/shortcuts/callback_shortcuts.0_test.dart:11]: `group` isn't necessary here.  As this test file only contains Callback Shortcuts tests.

File: examples/api/test/widgets/shortcuts/callback_shortcuts.0_test.dart [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      expect(
+        find.text('Press the down arrow key to subtract from the counter'),
+        findsOneWidget,
+      );
+      expect(find.text('count: 0'), findsOneWidget);
+    });
+
+    testWidgets(
+      'updates counter on arrowUp and arrowDown press',
+      (WidgetTester tester) async {
```
Comment (TahaTesser) [inline examples/api/test/widgets/shortcuts/callback_shortcuts.0_test.dart:31]:
```suggestion
      'updates counter on arrowUp and arrowDown press', (WidgetTester tester) async {
```

**Review by TahaTesser — CHANGES_REQUESTED** [review]

**Comment by ksokolovskyi** [comment]
@bleroux @TahaTesser thanks a lot for the review!

**Review by TahaTesser — APPROVED** [review]
LGTM!

**Pull request merged**